tests/XRootD/scitokens.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+function setup_scitokens() {
+	require_commands curl
+	if [ -z "$HOSTNAME" ]; then
+		HOSTNAME=`hostname`
+	fi
+
+	cat > "$PWD/scitokens/authdb" << EOF
+u * /issuer lr
+EOF
+	cat > "$PWD/scitokens/scitokens.cfg" <<EOF
+[Global]
+audience = https://$HOSTNAME:8095
+
+[Issuer test]
+issuer = https://$HOSTNAME:8095/issuer/one
+base_path = /protected
+EOF
+
+	# Setup the issuer and protected directory
+	ln -sf "$PWD/../issuer/export" "$PWD/scitokens/xrootd/issuer"
+	mkdir -p "$PWD/scitokens/xrootd/protected"
+	echo 'Hello, World' > "$PWD/scitokens/xrootd/protected/hello_world.txt"
+
+	# Override the scitoken cache location; otherwise, contents of prior test runs may be cached
+	export XDG_CACHE_HOME=$(mktemp -d "${PWD}/${NAME}/cache-XXXXXX")
+	mkdir -p "$XDG_CACHE_HOME/scitokens"
```

Comment:
The teardown procedure should always delete the scitokens directory (or the setup, which ensures a clean start), you can probably just set this as `export XDG_CACHE_HOME=${PWD}/${NAME}/cache`.

Reply:
Done.